.github/workflows/scripts/build-images.sh

```diff
@@ -16,5 +16,6 @@ for target in $@
 do
   DIRNAME="$(dirname "$target")"
   NAME="$(basename "$DIRNAME")"
-  make BUILD_IN_CONTAINER=false PUSH_MULTIARCH_TARGET="type=oci,dest=$OUTPUT/$NAME.oci" PUSH_MULTIARCH_TARGET_DISTROLESS="type=oci,dest=$OUTPUT/$NAME\-distroless.oci" push-multiarch-$target
+  make BUILD_IN_CONTAINER=false PUSH_MULTIARCH_TARGET="type=oci,dest=$OUTPUT/$NAME.oci" PUSH_MULTIARCH_TARGET_DISTROLESS="type=oci,dest=$OUTPUT/$NAME\-distroless.oci" \
+  PUSH_MULTIARCH_TARGET_CONTINUOUS_TEST="type=oci,dest=$OUTPUT/$NAME\-continuous\-test.oci" push-multiarch-$target
```

Comment:
Nit: let's make it readable by splitting all lines maybe?
```suggestion
  make \
    BUILD_IN_CONTAINER=false \
    PUSH_MULTIARCH_TARGET="type=oci,dest=$OUTPUT/$NAME.oci" \
    PUSH_MULTIARCH_TARGET_DISTROLESS="type=oci,dest=$OUTPUT/$NAME\-distroless.oci" \
    PUSH_MULTIARCH_TARGET_CONTINUOUS_TEST="type=oci,dest=$OUTPUT/$NAME\-continuous\-test.oci" \
    push-multiarch-$target
```